src/nopoll_decl.h

```diff
@@ -168,13 +168,17 @@
 /* additional includes for the windows platform */
 
 /* _WIN32_WINNT note: If the application including the header defines
- * the _WIN32_WINNT, it must include the bit defined by the value
- * 0x400. */
+ * the _WIN32_WINNT, it must include the bits defined by the value
+ * 0x501. */
```

Comment:
In MinGW for Windows, function declarations of getaddrinfo and freeaddrinfo are guarded by an "#if _WIN32_WINNT >= 0x501" in ws2tcpip.h. Surprisingly, they are not guarded in MinGW for Linux.